### DISCUSSION TITLE
large hours_to_show and display

### DISCUSSION THREAD

**Discussion by killerwahn** [dylandoamaral/uptime-card#167] [Q&A]
I would like to have an overview of a whole year with the uptime card.
If I set hours_to_show to 8760 (24h * 365d), "8760 hours ago" is displayed under the bar instead of "1 year ago". How can I change this?
![1](https://github.com/dylandoamaral/uptime-card/assets/149972900/6791bf9c-56aa-4d7b-b7cc-8007afc42c86)

**Comment by dylandoamaral** [comment]
HI, you can't with the current version of the card, feel free to raise an issue as a feature request.
